README.md

```diff
@@ -1,2 +1,30 @@
-# ios-sdk-spm
-Flybits Concierge SDK iOS Swift Package Manager
+# Flybits iOS SDK Swift Package
+
+Install Flybits iOS SDK easily with Swift Package Manager.
+
+## Installation
+
+Add the URL of this repository to your Xcode Project.
+
+Go to *File* > *Swift Packages* > *Add Package Dependency*, and paste in the link to this repository:
+
+`https://github.com/flybits/ios-sdk-spm.git`
+
+## Libraries
+
+There are 4 libraries:
+
+- Core
+- Concierge
+- Location
+- SmartRewards
+
+Here is the table for what are the SDKs inside each Library.
+
+| Core | Location | Concierge | SmartRewards |
+| --- | --- | --- | --- |
+| - FlybitsSDK<br/>- FlybitsPushSDK<br/>- FlybitsKernelSDK<br/>- FlybitsContextSDK | - FlybitsSDK<br/>- FlybitsContextSDK<br>- FlybitsContextLocationPluginSDK | - FlybitsSDK<br/>- FlybitsPushSDK<br/>- FlybitsKernelSDK<br/>- FlybitsContextSDK<br>- FlybitsConciergeSDK | - FlybitsSDK<br/>- FlybitsPushSDK<br/>- FlybitsKernelSDK<br/>- FlybitsContextSDK<br/>- FlybitsConciergeSDK<br/>- FlybitsSmartRewardsSDK |
```

Comment:
doesnt look like a table

Reply:
Yeah. it need to go to https://github.com/flybits/ios-sdk-spm/tree/readme-update for check how it looks like